File: python/6 kyu/276.py
# https://www.codewars.com/kata/5977ef1f945d45158d00011f/train/python

# Passed

def sep_str(st): 
    words = st.split(' ')
    max_length = max([len(w) for w in words])
    result = [[] for _ in range(max_length)]
    for col_idx, word in enumerate(words):
        for row_idx in range(max_length):
            if row_idx < len(word):
                elem = word[row_idx]
            else:
                elem = str()
                
            result[row_idx].append(elem)
            
    return result

output = sep_str("The Mitochondria is the powerhouse of the cell")
print(output)

File: nodejs/7 kyu/276.js
// https://www.codewars.com/kata/62a933d6d6deb7001093de16/train/javascript

// Passed

function getTheVowels(word) {
  let vowels = ["a", "e", "i", "o", "u"];
  let currentIdx = 0;
  let consecutiveVowelCount = 0;
  for (const letter of word) {
    if (letter === vowels[currentIdx]) {
      consecutiveVowelCount++;
      currentIdx++;

      if (currentIdx >= vowels.length) {
        currentIdx = 0;
      }
    }
  }

  return consecutiveVowelCount;
}

const output = getTheVowels("akfheujfkgiaaaofmmfkdfuaiiie");
console.log(output);